include/ddc/detail/kokkos.hpp

```diff
@@ -11,7 +11,7 @@
 
 #include <Kokkos_Core.hpp>
 
-#include "macros.hpp"
+#include "ddc/detail/macros.hpp"
```

Comment:
Not sure this change is sound. Using `"macros.hpp"` will make the compiler look for a macros.hpp file in the current directory and not to browse the whole path list of includes.

Reply:
"source": https://blogs.stonesteps.ca/1/p/64